tarodb/sqlite/queries/assets.sql

```diff
@@ -14,10 +14,12 @@ INSERT INTO asset_minting_batches (
 ) VALUES (0, ?, ?);
 
 -- name: FetchMintingBatchesByInverseState :many
-SELECT *
+SELECT batches.*, keys.*, txs.height_hint
 FROM asset_minting_batches batches
 JOIN internal_keys keys
     ON batches.batch_id = keys.key_id
+LEFT JOIN chain_txns txs
```

Comment:
Every batch will have a chain transaction associated with it, so I think we can do a regular join here?

Reply:
Also FWIW, for minting matches, the only time the chain transaction is inserted is once the PSBT has been signed. At that point, we insert a chain txn, then associate that with a managed_utxo for all the assets in the batch to anchor them. 

At that point `CommitSignedGenesisTx`, we want to also insert a height hint there.

Reply:
After calling `NewMintingBatch` there won't be a `chain_tx` available. Only after `CommitSignedGenesisTx` as you mention, the tx will be inserted and associated with the batch. This is also when we insert the height hint.